### DISCUSSION TITLE
Is Promxy act as a LB ?

### DISCUSSION THREAD

**Discussion by sudhir-maurya** [jacksontj/promxy#419] [Q&A] [answered]
Does Promxy functions as a load balancer also for Prometheus HA setup for query ?

**Comment by jacksontj** [comment, marked as answer]
I'm not quite sure what you meant to ask here. Promxy is an aggregating proxy; so it aggregates data from downstream prom APIs to provide a unified API. So it "load balances" in the sense that it routes the request to the servers required to answer the API call; but as of now it sends the traffic to all servers that match the data. There have been some ideas kicked around for reducing that (https://github.com/jacksontj/promxy/issues/2) but in practice this has been a pretty minor issue.
